src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -86,7 +90,15 @@ const ipVersionOptions = ref([
 ])
 
 const recordOptionsButCurrent = computed(() => {
-  return props.recordOptions?.filter((option) => option.id !== props.currentHostSet?.id)
+  // Filter out objects from recordOptions based on the presence of an IP address with a hyphen in allObjects
+  const objectsWithHyphenIp = props.allObjects
```

Comment:
This way we are preventing the creation of host sets containing IP ranges, regardless they presence of port forwards. A more correct implementation might be:
- While editing a host set, determine if the current host set is being used by a port forward
- If current host set is used by a port forward, then we need to track which objects contains IP ranges directly or indirectly, by looping `ns.objects get-info` until there are no more object references :confused:
- Finally, we should remove these objects from the Records combobox

Reply:
so I based my work on a bug of the UI, actually when you have an object that use another object it is not available to be used as a portforward object

Inevitably searching for a sub-sub-sub-object that has a parent object using port forwarding makes the work much more complicated, searching only for the direct object is simpler.